sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxDocumentClientImpl.java

```diff
@@ -2360,8 +2366,6 @@ private Mono<ResourceResponse<Document>> readDocumentInternal(String documentLin
 
             CosmosEndToEndOperationLatencyPolicyConfig endToEndPolicyConfig = getEndToEndOperationLatencyPolicyConfig(options);
 
-            request.requestContext.setEndToEndOperationLatencyPolicyConfig(endToEndPolicyConfig);
```

Comment:
why did we remove this?

Reply:
It is moved to getRxDocumentServiceResponseMonoWithE2ETimeout as it is required for all point operatoins

Reply:
Got it, thanks!